database.py

```
@@ -0,0 +1,222 @@
+import aiosqlite
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)